Formula/xalan-c.rb

```diff
@@ -20,11 +21,16 @@ class XalanC < Formula
   end
   depends_on "xerces-c"
 
+  needs :cxx11
+
   # Fix segfault. See https://issues.apache.org/jira/browse/XALANC-751
+  # Build with char16_t casts.  See https://issues.apache.org/jira/browse/XALANC-773
   patch :DATA
 
   def install
+    ENV.cxx11
     ENV.deparallelize # See https://issues.apache.org/jira/browse/XALANC-696
+    ENV["CXXFLAGS"] = "-std=c++11"
```

Comment:
this shouldn't be needed now that you've added ENV.cxx11.

Reply:
D'oh.  I'll push that in just a second.